packages/console/src/components/RoleScopesTransfer/components/SourceScopesBox/index.tsx

```diff
@@ -129,14 +132,15 @@ const SourceScopesBox = ({ roleId, selectedScopes, onChange }: Props) => {
           onChange={handleSearchInput}
         />
       </div>
-      <div className={transferLayout.boxContent}>
-        {!isLoading && !hasError && dataSource.length === 0 && (
+      <div
+        className={classNames(transferLayout.boxContent, isEmpty && transferLayout.emptyBoxContent)}
```

Comment:
This may be not necessary. I think the problem is the `.empty` div in `<DataEmpty />` does not take 100% height of the `.boxContent` div.
So you need to make it take 100% height and then use `justify-content: center` to vertical align the image

Reply:
Good call, updated.